.github/workflows/docker.yml

```diff
@@ -28,23 +28,19 @@ jobs:
           export commit_id=$(git log --format="%H" -n 1)
           echo "commit id = $commit_id"
           echo "COMMIT_ID=$commit_id" >> $GITHUB_ENV
-      - name: Get lmdeploy version
-        if: startsWith(github.ref, 'refs/tags/') == false
+      - name: Build and push the latest Docker image
         run: |
-          echo "TAG=$TAG_PREFIX:latest"  >> $GITHUB_ENV
-      - name: Get lmdeploy tag
+          export TAG=$TAG_PREFIX:latest
+          echo $TAG
+          docker build docker/ -t ${TAG} --build-arg COMMIT_ID=${COMMIT_ID}
+          docker push $TAG
+          echo "TAG=${TAG}" >> $GITHUB_ENV
+      - name: Push docker image with released tag
         if: startsWith(github.ref, 'refs/tags/') == true
         run: |
           export LMDEPLOY_VERSION=$(python3 -c "import sys; sys.path.append('lmdeploy');from version import __version__;print(__version__)")
           echo $LMDEPLOY_VERSION
-          echo "TAG=${TAG_PREFIX}:v${LMDEPLOY_VERSION}" >> $GITHUB_ENV
-      - name: Build Docker image
-        continue-on-error: true
-        run: |
-          echo $TAG
-          docker build docker/ -t ${TAG} --build-arg COMMIT_ID=${COMMIT_ID}
-      - name: Push Docker image
-        continue-on-error: true
-        run: |
-          echo $TAG
-          docker push $TAG
+          export RELEASE_TAG=${TAG_PREFIX}:v${LMDEPLOY_VERSION}
```

Comment:
Will the "push docker image with released tag"  be intriggerd after "lmdeploy:latest" image is built?

Reply:
No. It will be triggered only if this line is met:`if: startsWith(github.ref, 'refs/tags/') == true`